j-db-client/prisma/schema.prisma

```diff
@@ -8,39 +8,40 @@ generator prisma_client {
 }
```

Comment:
@Lanny - Prisma seems to prefer declaring `@id` first so any one-line changes simply switching that are just to satisfy the Prisma schema formatter plugin. Same for whitespace changes.